### DISCUSSION TITLE
Cockpit - LoginTo / LoginTitle - Unable to Change

### DISCUSSION THREAD

**Discussion by marcoczen** [cockpit-project/cockpit#20219] [Q&A] [answered]
Hi. 

From https://cockpit-project.org/guide/latest/cockpit.conf.5.html,

``` 
LoginTitle - Set the browser title for the login screen.

LoginTo - When set to false, direct remote logins are disallowed. 
```

I tried adding these 2 directives ( OpenSuse 15.5  - Cockpit 309.x  ) in;
```
/etc/systemd/system/cockpit.socket.d/cockpit.conf

```
but i get the error message ;

```
/etc/systemd/system/cockpit.socket.d/cockpit.conf:1: Unknown section 'WebService'. Ignoring.

```

I have tried removing [WebService] but then it says `Assignment outside of section. Ignoring.`

Anyone  knows the correct syntax / correct location to put these 2 directives in ?

Thanks.

**Comment by marusak** [comment, marked as answer]
The first sentance of the doc you links says `Cockpit can be configured via /etc/cockpit/cockpit.conf.`. Why do you try adding it into `/etc/systemd/system/cockpit.socket.d/cockpit.conf` and not in `/etc/cockpit/cockpit.conf`?

**Comment by martinpitt** [comment]
>  /etc/systemd/system/cockpit.socket.d/cockpit.conf

This is a *systemd* configuration file. You want to create /etc/cockpit/cockpit.conf, see the [manpage.](https://cockpit-project.org/guide/latest/cockpit.conf.5.html)
